### DISCUSSION TITLE
using the gh api in a github action to trigger something for a different repo

### DISCUSSION THREAD

**Discussion by hanbollar** [community/community#33529] [Apps, API and Webhooks] [answered]
hiya ~

currently have the following setup

- repoA: (private)
  - github action
- repoB: (public)
  - issue templateB
- repoC: (public)
  - issue templateC

trying to have it so that the github action in repoA can trigger the creation of new issues in repoB and repoC using the associated templates

using git directly in the action and using gh directly in the action does the following:
- it is able to get the templates from the other repos
- it can only create new issues in repoA using the `gh issue create` setup
- even when i use the `--repo` flag inside the action to tell `gh` what repo to create the issue in, it still just creates it in repoA
- even when in the github action i manually git clone the other repos and cd into them to use the `gh issue create` it still just creates the new issues in repoA

question:
- does gh issue create only work in the current repo it's in when it's used in a github action? or is there a piece of syntax im missing somewhere?

tia!

**Comment by airtower-luna** [comment, marked as answer]
What kind of token do you use for `gh`? If it's the `GITHUB_TOKEN` provided by the workflow then yes, it's limited to the current repository.

If you want to create issues in other repositories you'll need to [create a PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with appropriate permissions and use that instead.

**Reply by hanbollar** [reply]
oh sweet, that worked, tysm!
